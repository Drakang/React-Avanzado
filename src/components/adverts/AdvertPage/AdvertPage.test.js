import AdvertPage from './AdvertPage'
import React from 'react';
import { render} from '@testing-library/react';
import { Provider } from 'react-redux';

describe('AdvertPage', () => {
  test('snapshot test', () => {
    const store = {
      getState: () => ({
        ui: {},
        adverts: {
          tags: [],
          getById: {
            name: '',
            sale: true,
            price: 0,
            tags: [],
          },
        },
      }),
      subscribe: () => {},
      dispatch: () => {},
    }
    const root= () => 
    render(
      <Provider store={store}>
        <AdvertPage />
      </Provider>
    );


    expect(root).toMatchSnapshot()
    })
})
