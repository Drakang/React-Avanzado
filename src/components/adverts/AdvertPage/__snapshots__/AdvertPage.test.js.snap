// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdvertPage snapshot test 1`] = `[Function]`;
